.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -31,7 +31,7 @@ Would a visual be helpful for reviewers? e.g. "Before" and "After", visual chang
 
 ## Checklist
 
-- [ ] If any new components were added, there are exported from `packages/index.ts`
```

Comment:
Opted to remove this one because we rarely add new components, and this is a common step when working with file exports in React.